apps/api/src/app/environments/usecases/create-environment/create-environment.e2e.ts

```diff
@@ -24,8 +25,13 @@ describe('Create Environment - /environments (POST)', async () => {
     expect(body.data.identifier).to.be.ok;
     const dbApp = await environmentRepository.findOne({ _id: body.data._id });
 
+    if (!dbApp) {
+      throw new Error('App not found');
+    }
```

Comment:
Actually, this is needed only in order to remove the ts errors and should never occur 😅

however, i do agree that his code could be reused/copied in other places due to missing knowledge of its purpose, in order to prevent it i can do the following.

```
 if (!dbApp) {
      expect(dbApp).to.be.ok;
      throw new Error('App not found');
    }
```

wdyt?

Reply:
It still feels a little unnatural due to throwing an error after an assertion, but I think it's a suitable compromise.